tests/tests/test_monitor_client.py

```diff
@@ -159,7 +159,7 @@ def prepare_env(self, env, user_name):
         devauth_tenant = DeviceAuthV2(auth)
 
         env.new_tenant_client("configuration-test-container", tenant["tenant_token"])
-        mender_device = MenderDevice(env.get_mender_clients()[0])
+        env.device = mender_device = MenderDevice(env.get_mender_clients()[0])
```

Comment:
why exactly do we need this? (I mean env.device assignment)

Reply:
For our hook to find the device on errors. See [these lines](https://github.com/mendersoftware/integration/blob/4d4c46dcc29e713a1d0ac4278d674509f4ec15ea/tests/conftest.py#L121-L139).